{
  "_annotation": {
    "cve_id": "CVE-2024-26149",
    "reason": "Added CPE configurations because not yet analyzed by NVD.",
    "snapshot": "https://raw.githubusercontent.com/anchore/nvd-data-overrides/main/.snapshot/2024/CVE-2024-26149.json",
    "triage_notes": {
      "references": [
        "https://github.com/vyperlang/vyper/security/advisories/GHSA-9p8r-4xp4-gw5w"
      ]
    }
  },
  "cve": {
    "configurations": [
      {
        "nodes": [
          {
            "cpeMatch": [
              {
                "criteria": "cpe:2.3:a:vyperlang:vyper:*:*:*:*:*:python:*:*",
                "matchCriteriaId": "832C489D-4288-46B4-A29E-0E7168748042",
                "versionEndIncluding": "0.3.10",
                "vulnerable": true
              }
            ],
            "negate": false,
            "operator": "OR"
          }
        ]
      }
    ]
  }
}